lib/susedistribution.pm

```diff
@@ -190,15 +190,16 @@ sub script_sudo($$) {
     return;
 }
 
+# simplified but still colored prompt for better readability
 sub set_standard_prompt {
     my ($self, $user) = @_;
     $user ||= $testapi::username;
     if ($user eq 'root') {
         # set standard root prompt
-        type_string "PS1='# '\n";
+        type_string "PS1=\"\$(tput bold 2; tput setaf 1)#\$(tput sgr0) \"\n";
     }
     else {
-        type_string "PS1='\$ '\n";
+        type_string "PS1=\"\$(tput bold 2; tput setaf 1)\\\$\$(tput sgr0) \"\n";
```

Comment:
no. I just brought back the default color from SUSE/openSUSE to make the prompt distinguishable from the entries. The character '$' and '#' are IMHO distinguishable enough unless someone tries to cover them in a full screen needle but no single character will be different enough in that context.